### ISSUE TITLE
Max size of photo

### ISSUE THREAD

**Issue by Esysteme** [Chevereto/Chevereto-Free#75]
There is an internal limit to 2M on free version ?

On upload : 

![Capture d’écran de 2021-02-08 19-05-24](https://user-images.githubusercontent.com/2485014/107262453-dd688d80-6a40-11eb-80c2-f6fc7c1fec24.png)

On dashboard : 

![Capture d’écran de 2021-02-08 19-05-37](https://user-images.githubusercontent.com/2485014/107262494-e9ece600-6a40-11eb-9db8-b76d80468c81.png)

I miss something ?

**Comment by rodolfoberrios** [comment]
https://v3-docs.chevereto.com/setup/server-issues.html#recommended-php-ini

**Comment by Esysteme** [comment]
Ok but i don't undertand why 2M in front, I reinstall and now limit is 10M, and i have not idea where come from this number

**Comment by rodolfoberrios** [comment]
Dear user, kindly check the docs:

https://v3-docs.chevereto.com/settings/image-upload.html#maximum-upload-file-size-mb
https://v3-docs.chevereto.com/setup/server-issues.html#recommended-php-ini

As you can check, the upper limit is capped by `php.ini` and the internal limit (front end) is set on the dashboard.

You also get a hint about this directly when editing the dashboard:

![image](https://user-images.githubusercontent.com/20590102/107663104-c35dc380-6c69-11eb-9515-e269c82a74d7.png)

Most likely you skipped the part where I explain you what is this `php.ini` file. In case you ignore it, is **your server** which handle these settings, and I even provide you with a method to use my layer to override that:

https://v3-docs.chevereto.com/setup/settings-file.html#runtime-php-ini-configuration

Bottom line, do please read and do not use this issue section for non-code issues.
